extensions/vscode/package.json

```diff
@@ -119,10 +138,6 @@
     ]
   },
   "scripts": {
-    "__esbuild-base": "esbuild ./src/extension.ts --bundle --outfile=dist/extension.js --external:vscode --format=cjs --platform=node",
-    "__dev": "pnpm run esbuild-base -- --sourcemap --watch",
-    "__vscode:prepublish": "pnpm run esbuild-base -- --minify",
-    "__build": "vsce package",
```

Comment:
I removed those because I don't think we should use them? @sulco what do you think?